frontend/src/screens/auth.css

```
@@ -762,60 +791,65 @@
- * being squeezed into fixed columns, so a wrapped one is the exception; if a
- * translation makes it the rule, the row is what changes, in one place.
+ * What that costs, stated: a sentence long enough to wrap gets its badge beside
+ * the MIDDLE of the paragraph rather than beside its first line. Two lines is the
+ * shape this is tuned for; a translation that needs three is what would change
+ * the row, in one place.
  */
-.auth-limit-icon {
-  --limitIcon: 22px;
+.auth-promise-icon {
+  --promiseIcon: 22px;
   flex: none;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)